core/web/presenters/terra_chain.go

```diff
@@ -0,0 +1,36 @@
+package presenters
+
+import (
+	"time"
+
+	terratypes "github.com/smartcontractkit/chainlink/core/chains/terra/types"
+)
+
+// TerraNodeResource is a Terra node JSONAPI resource.
+type TerraNodeResource struct {
+	JAID
+	Name          string    `json:"name"`
+	TerraChainID  string    `json:"terraChainID"`
+	TendermintURL string    `json:"tendermintURL"`
+	FCDURL        string    `json:"fcdURL"`
+	CreatedAt     time.Time `json:"createdAt"`
+	UpdatedAt     time.Time `json:"updatedAt"`
+}
+
+// GetName implements the api2go EntityNamer interface
+func (r TerraNodeResource) GetName() string {
```

Comment:
Maybe a more appropriate fn name could be `GetType()`. wdyt?

Don't have full ctx, but `r.Name` vs `r.GetName` feels confusing.

Reply:
I think we're bound by the api2go types here.